Userland/Utilities/strings.cpp

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/CharacterTypes.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+#define DEFAULT_MINIMUM_STRING_LENGTH 4
```

Comment:
Is there a point in making that a `#define`? It's only used for initializing the variable, and that is already named `minimum_string_length`.

Reply:
My intent was to adhere to the existing code style used by other utilities in the system, e.g. `tail` has `DEFAULT_LINE_COUNT` which is used to initialize `wanted line count`. I can change this if necessary.

Reply:
That particular part of `tail` has been there since 2019, so it doesn't necessarily reflect whatever has been established since then. Usually, as long as it has _any_ name that makes sense (and it's not just some constant that's _somewhere_ in the code), we're fine with it. No point in making a separate define just for the purpose of initializing a single variable.

Reply:
Ok, removed in [2fa4b20](https://github.com/SerenityOS/serenity/pull/16013/commits/2fa4b20928b0345022e54fdff7a67319cffa6e5a)